docs/security/security.rst

```diff
@@ -1,32 +1,36 @@
 .. _security_with_docker :
 
-Security Overview
-=================
+Docker Security
+===============
 
-Using security is optional - non-secured clusters are supported, as well as a mix of authenticated, unauthenticated, encrypted and non-encrypted clients.  The following security features are currently supported on the Confluent Platform Docker images:
+Using security is optional. Confluent Platform supports non-secured clusters and a mix of authenticated, unauthenticated,
```

Comment:
It might read better to start with the secure options and then listing the less secure in the order of security.

Reply:
@edenhill do you mean something like this? `Confluent Platform supports cluster encryption and authentication, including a mix of authenticated and unauthenticated, and 
encrypted and non-encrypted clients. Using security is optional.`?

Reply:
Yep, that's more positive :)